packages/skills/carpark_detection/handlers.py

```diff
@@ -236,9 +236,9 @@ def _handle_inform(self, msg: FIPAMessage, sender: str, message_id: int, dialogu
             tx_digest = json_data['transaction_digest']
             logger.info("[{}]: checking whether transaction={} has been received ...".format(self.context.agent_name,
                                                                                              tx_digest))
-            proposal = cast(Description, dialogue.proposal)
-            total_price = cast(int, proposal.values.get("price"))
-            is_settled = self.context.ledger_apis.is_tx_settled('fetchai', tx_digest, total_price)
+            # proposal = cast(Description, dialogue.proposal)
```

Comment:
Please revert. The seller needs to check if the transaction references the correct amount.